### ISSUE TITLE
Tensorboardx freezing without errors

### ISSUE THREAD

**Issue by jandevries123** [lanpa/tensorboardX#532]
I am experiencing crashes/freezing in my keras code. The code freezes without an error, it just stops. 

Code i am using

`self.writer = SummaryWriter("Tensy")`
`self.writer.add_scalars('Softmax', {"Action 1" : predicted_action[0][0], "Action 2" : predicted_action[0][1], "Action 3" : predicted_action[0][2]}, self.gstep)	`

I've calculated that it is around the flush time that it crashes wheter its the first flush round or later in the run... nonetheless it crashes every run, making my run proccess unreliable to do a long run. If it succeeds to save, then there is nothing wrong with the file itself. Are there any known issues of tensorboardx freezing code? I didn't want to report this as a bug, because it might just me making some mistake.

Hope someone can help

**Comment by lanpa** [comment]
@jandevries123 Will it still freeze if you change `predicted_action[0][x]` to a scalar and `self.gstep` to a increasing scalar?
